index.html

```diff
@@ -46,8 +46,6 @@
     </script>
     <script src="schema.js"></script>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
-    <!-- <meta name="google-site-verification" content=""/> -->
```

Comment:
not needed anymore, google domains automatically verifies us as the owner of the domain